test/test_wait.py

```diff
@@ -14,22 +16,25 @@
     wait_for_write,
 )
 
+_TYPE_SOCKET_PAIR = Tuple[socket, socket]
```

Comment:
Take it or leave it: since tests don't need to worry about privacy, the `_` is not really necessary.